src/components/Player/Pages/Heroes/playerHeroesColumns.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import { Link } from 'react-router';
 import { transformations, getPercentWin } from 'utility';
 import { TablePercent } from 'components/Visualizations';
 import strings from 'lang';
 
-export const playerHeroesOverviewColumns = [{
+export const playerHeroesOverviewColumns = playerId => [{
   displayName: strings.th_hero_id,
   tooltip: strings.tooltip_hero_id,
   field: 'hero_id',
-  displayFn: transformations.hero_id,
+  displayFn: row => <Link to={`/players/${playerId}/matches?hero_id=${row.hero_id}`}>{transformations.hero_id(row)}</Link>,
```

Comment:
No, I don't think so. There is no reason to add new column when we can make hero clickable.